ruby/spec/authentication_spec.rb

```diff
@@ -14,14 +14,7 @@
 
   context 'testing health endpoint for authentication' do
 
-    credentials = []
-    File.foreach('config/.access') do |line|
-      credentials.append(line.split('=', 2)[1].strip)
-    end
-
-    token = Base64.encode64("#{credentials[0]}:#{credentials[1]}")
-
-    let(:response) { get '/', nil, {'HTTP_AUTHORIZATION' => "Basic #{token}" } }
+    let(:response) { get '/', nil, {'HTTP_AUTHORIZATION' => "Basic #{create_token}" } }
```

Comment:
Can we make a method out of `{'HTTP_AUTHORIZATION' => "Basic #{create_token}" }`?

So basically

```suggestion
    let(:response) { get '/', nil, auth_header }
```

```ruby
def auth_header
  {'HTTP_AUTHORIZATION' => "Basic #{auth_header_content}" }
end
```

Reply:
Yes, that looks more elegant and I will change it. But I hate how in Ruby a method looks like a variable - it gives a slightly wrong impression